### PR TITLE
Add skip environment tags for link text that is being deployed

### PR DESCRIPTION
**Pull request by idavidmcdonald**
This is the supplier frontend PR that broke the tests - https://github.com/alphagov/digitalmarketplace-supplier-frontend/pull/714

### CODE REVIEW

**Comment by benvand** [comment]
Should the skip tags be the other way around?
The older, existing functionality will be on staging, and should therefore be skipped on preview.
The newer test, with the functionality we're introducing, will be available on preview only and so we need to skip it on staging.

**Comment by idavidmcdonald** [comment]
I think it is correct as it stands?

At the moment on preview we have the text 'View services'
At the moment on staging we have the text 'View'

**Comment by MuriloDalRi** [comment]
The test looks good 👍

File: features/supplier/view_and_edit_services.feature [hunk 1]
```diff
@@ -24,3 +25,22 @@ Scenario Outline: Supplier coming from dashboard to view the detail page for one
     | digital-outcomes           | Digital outcomes           | Team capabilities           | Agile coaching   |
     | user-research-participants | User research participants | Recruitment approach        | Entirely offline |
     | user-research-studios      | GDSvieux Innovation Lab    | Lab address                 | GDSbury          |
+
+@skip-preview
```
Comment (samuelhwilliams) [inline features/supplier/view_and_edit_services.feature]: Be nice to throw in `@skip-local` as well, for anyone who tries to run the functional tests locally.

**Pull request merged**